bionic/flow.py

```diff
@@ -844,46 +844,81 @@ def to_builder(self):
 
         return FlowBuilder._from_state(self._state)
 
-    def get(self, name, fmt=None):
+    def get(self, name, collection=None, fmt=None, mode=object):
         """
         Computes the value(s) associated with an entity.
 
-        If the entity has multiple values, the ``fmt`` parameter indicates
-        how to handle them.  It can have any of the following values:
+        If the entity has multiple values, the ``collection`` parameter
+        indicates how to handle them.  It can have any of the following values:
 
-        * ``object``: return a single value or throw an exception
+        * ``None``: return a single value or throw an exception
         * ``list`` or ``'list'``: return a list of values
         * ``set`` or ``'set'``: return a set of values
         * ``pandas.Series`` or ``'series'``: return a series whose index is
           the root cases distinguishing the different values
 
+        The user can specify the type of object (implicitly specifying in-memory vs. persisted data)
+        to return in the collection using the ``mode`` parameter.  It can have any of the
+        following values:
+        * ``object`` for a value in-memory
+        * ``FileCopier`` for a wrapper for a path to the persisted file for the computed entity
```

Comment:
Shouldn't this have quotes around it, since we expect a string?

(Sorry for missing this the first time.)